docs/setup/pip.md

```diff
@@ -16,7 +16,7 @@ We support :simple-python: **Python from 3.8 to 3.12**.
     wget -O "$TEMP_DEB" "https://github.com/KhiopsML/khiops/releases/download/10.2.0/khiops-core_10.2.0-1-${CODENAME}.amd64.deb" && \
```

Comment:
Change "10.2.0" to "10.2.1", even though the latter is still a pre-release. But as per the agreed versioning scheme, the current khiops-python (10.2.1.0) depends on Khiops (core) 10.2.1.

Reply:
My understanding was we stay on the 10.2.0 as long as it is the latest official release (the reason why @folmos-at-orange added the Rocky Linux 8 binary to this 10.2.0 release yesterday). 
I'm not comfortable linking the installation to a prerelease.

Reply:
Indeed is the 10.2.0, the 10.2.1 wasn't "released". To my knowledge this will be the last "shadow" release. 

Because of a discoordination we shipped the rocky 8 patch 10.2.1.0. It is a minimal version mismatch and it will eventually dissapear.

Reply:
OK, so, I my understanding, we have:

- Khiops the binaries (`khiops-core` OS package: 10.2.0)
- Khiops Python (Conda, Pip): 10.2.1.0 which depends on Khiops the binaires 10.2.0.

Is this correct?

Reply:
Yes